src/crosshook-native/src/styles/host-tool-dashboard.css

```
@@ -0,0 +1,543 @@
+
+.crosshook-host-tool-dashboard-tile__header {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.crosshook-host-tool-dashboard-tile__dot {
+  width: 10px;
+  height: 10px;
+  border-radius: 999px;
+  background: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)